### DISCUSSION TITLE
Sort & Filter using custom fields (e.g. sort: creation_time-asc updated:yyyy-mm-dd..yyyy-mm-dd))

### DISCUSSION THREAD

**Discussion by miaits** [community/community#14888] [Projects and Issues]
Query could look like this `is:issue is:open sort:creation_time-asc updated:2022-01-01..2022-03-31 `

Use Cases: 
- View only issues that were updated in the x days/weeks/months within a specific project. 
- Sort issues by last update using `sort:updated-desc`

**Comment by tlabeeuw** [comment]
This is critical for us. We want to be able to get under the [1,000 limit](https://docs.github.com/en/graphql/reference/queries#search)

```graphql
query GetIssues($cursor: String, $search: String!) {
  search(query: $search, type: ISSUE_ADVANCED, after: $cursor, first: 100) {
    issueCount
    pageInfo {
      hasNextPage
      endCursor
    }
    nodes {
      ... on Issue {
        id
        number
     }
  }
}
```

With a variables like:
```json
{
  "search": "repo:user/repo is:issue has:customField customField:\"value\""
}
```
Without this how do we get around the limit? Either increase the limit to let us do client side filtering, or add the ability to do more custom filtering.

A good example would be a `Status` field which has states `Backlog`, `In progress`, `In review`, `Accepted`, etc. Then searching by `status:"In review"`, allows us to easily get under the limit.
